src/App.tsx

```diff
@@ -1,39 +1,61 @@
-import React from 'react';
```

Comment:
Instead of importing `Component` separately, you can import it along with `React`. For example, `import React, { Component } from 'react'`.